tests/test_web/test_s3_utils.py

```diff
@@ -13,10 +13,10 @@ def test_get_object_from_s3_bucket(mock_boto3):
     my_mock.meta.client.get_object.return_value = {
         'ResponseMetadata': {
             'RequestId': 'CK2VG4V5ZQXAAM5B',
-            'HostId': 'q4Wp/tsvjnl/eBeN0dHvHYi6xUl9U149BdN6IAXjaFJnnQX+mLjg8hM4xAfgijfDo3fugYSEpPA=',
+            'HostId': 'q4Wp/tsvjnl/eBeN0dHvHYi6xUl9U149BdN6IAXjaFJnnQX+mLjg8hM4xAfgijfDo3fugYSEpPA=',  # noqa: E501
```

Comment:
Since this is just mocked return value for test, can not we just make it shorter instead of disabling the check?

Reply:
@yashvardhannanavati what do you think?

Reply:
Sure, let's make it short

Reply:
Ok!

Reply:
Pushed again, please let me know if it's good